feature-libs/cart/saved-cart/components/details/saved-cart-details-items/saved-cart-details-items.component.html

```diff
@@ -1,6 +1,10 @@
 <ng-container *ngIf="savedCart$ | async as cart">
   <ng-container *ngIf="cart?.entries?.length > 0; else emptyCartItems">
-    <div role="status" [attr.aria-label]="'common.loaded' | cxTranslate"></div>
+    <div
+      *cxFeature="'!a11yRemoveStatusLoadedRole'"
```

Comment:
MockFeatureDirective should be added to TestingConfiguration to avoid error in tests logs "cxFeature not known element of div.."

Applicable for all changed files where cxFeature was introduced